### DISCUSSION TITLE
Having something over an active dialog will select first element in dialog

### DISCUSSION THREAD

**Discussion by pauubach** [vuetifyjs/vuetify#20519] [Q&A]
Hello, 
I have a webapp in Vue 2.7 and Vuetify 2.4. I am using delighted, which displays a survey on the web. 
So, it shows a box in front of my content. 

When I have a dialog opened, and the survey appears, selecting anything in the survey selects the first element (button, input field, ...) in the dialog, and deselects the survey. So, there's no way I can send comments in the survey without closing the dialog. 

In order to test this behavior, to don't have to install delighted, you can open the persisten dialog in Vuetify 2: 
https://v2.vuetifyjs.com/en/components/dialogs/#persistent

In console, select the v-overlay and disable z-index, so it's not covering all and you can click other elements. If you click a button (View source, invert colors of an example...), you'll see how dialog first button is selected. It's not too bad, as the action is launched. But trying to select Search field on top is impossible. As soon as it is selected, focus is sent to first element in dialog, so you're unable to write anything there. 

I haven't been able to find any information regarding this issue, and it seems to be fixed in Vuetify 3. 

Have you ever experienced something similar or could anyone please provide some help/guidance?

Thanks,
Pau

**Comment by KaelWD** [comment]
https://v2.vuetifyjs.com/en/api/v-dialog/#props-retain-focus
